src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamsBase.java

```diff
@@ -143,32 +136,18 @@ public GetRecordsResult getRecords(GetRecordsRequest getRecordsRequest) {
             LOG.debug("getRecords request={}", getRecordsRequest);
```

Comment:
`getRecords` base class behavior change: instead of assuming we always need to filter and map, just determine whether the method is called with tenant context or without. The actual retrieving of records is delegated to the sub-class (currently either byTable or byIndex).